proto/tendermint/state/utils.go

```diff
@@ -25,7 +25,7 @@ func CopyFields(dst interface{}, src interface{}) {
 }
 
 func (m *ABCIResponses) ValidatorUpdates() types.ValidatorUpdates {
-	if m.EndBlock.ValidatorUpdates != nil {
+	if m.EndBlock != nil {
```

Comment:
@lyhe18 is it possible that `ValidatorUpdates` is nil while other fields in `m.Endblock` is not nil?

Reply:
yes, but it still works in this situation, the function can just return the nil validator update.

Reply:
ah I see